spec/controllers/shift_spec.rb

```diff
@@ -0,0 +1,144 @@
+require "rails_helper"
+
+RSpec.describe "Shifts", type: :request do
+  describe "GET /index" do
+    before do
+      FactoryBot.create_list(:shift, 10)
+      @shift = Shift.first
+      get "/api/v1/shifts"
+      @first_shift_data = json["data"][0]["attributes"]
+    end
+    
+    it "returns status code 200 (success)" do
+      expect(response).to have_http_status(:success)
+    end
+
+    it "returns all shifts" do
+      expect(json["data"].size).to eq(10)
+    end
+
+    it "returns correct shift_start" do
+      expect(Date.parse(@first_shift_data["shift_start"])).to eq(@shift.shift_start.strftime("%B %d, %Y").to_date)
+    end
+
+    it "returns correct shift_end" do
+      expect(Date.parse(@first_shift_data["shift_end"])).to eq(@shift.shift_end.strftime("%B %d, %Y").to_date)
```

Comment:
I think you can simplify that shift_end formatting to `@shift.shift_end.to_date` and still have it pass.  If so, the 3 other tests like this can be updated as well.